### ISSUE TITLE
vertex_builder has an unused third type argument

### ISSUE THREAD

**Issue by CryZe** [nical/lyon#526]
```rust
pub fn vertex_builder<OutputVertex, OutputIndex, Input, Ctor>(
    buffers: &mut VertexBuffers<OutputVertex, OutputIndex>, 
    ctor: Ctor
) -> BuffersBuilder<OutputVertex, OutputIndex, Ctor>
```

The type parameter `Input` is not used, which makes `vertex_builder` annoying to use, as Rust can't infer its type (as there is none), so you have to call `vertex_builder` like so:
```rust
vertex_builder::<_, _, (), _>(...)
```

Changing this would be a breaking change, so this would have to go into the next breaking release.

I'm surprised Rust didn't emit a warning for this? (Maybe it did and it was overlooked)

**Comment by nical** [comment]
Thanks for the report. This was overlooked when revamping the geometry building interface. Looks like this function is not used anywhere in the code base.

I'd like to postpone making a breaking release so I'll start by hiding this function from the docs (you can use `BuffersBuilder::new` in the mean time).

**Comment by nical** [comment]
Fixed in the master branch (0.16 not published yet).
